packages/reassure-measure/src/measure.tsx

```diff
@@ -1,15 +1,17 @@
 import * as React from 'react';
-import { cleanup, RenderAPI } from '@testing-library/react-native';
+import type { RenderResult as RTLRenderApi } from '@testing-library/react';
+import type { RenderAPI as RNTLRenderApi } from '@testing-library/react-native';
 import * as math from 'mathjs';
 import { config } from './config';
 import { showFlagsOuputIfNeeded, writeTestStats } from './output';
 import type { MeasureRenderResult } from './types';
 
-export interface MeasureOptions {
+interface MeasureOptions {
+  name?: string;
   runs?: number;
   dropWorst?: number;
   wrapper?: (node: React.ReactElement) => JSX.Element;
-  scenario?: (view?: RenderAPI) => Promise<any>;
+  scenario?: (view?: RNTLRenderApi | RTLRenderApi) => Promise<any>;
```

Comment:
1. I'd drop the optional from `view`. User should be certain we provide it.
2. I am not sure about using `RNTLRenderApi | RTLRenderApi`. Because a) we discourage using it in favor of screen, b). he will probably still need to do explicit typing in his own code. So maybe `any`?